Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -44,21 +45,29 @@ final class SettingsViewModel {
         }
     }
     
-    func addVehicle(_ vehicle: Vehicle) async {
-        
+    /// Adds a new vehicle to the Firestore database and the local `vehicles` array.
```

Comment:
```suggestion
    /// Adds a new vehicle to the Firestore database and the local ``SettingsViewModel/vehicles`` array.
```

Reply:
This makes it so that you can link to this property in the view model, in the documentation! See this when building the documentation
* Product > Build Documentation